.gitignore

```diff
@@ -3,7 +3,8 @@ dist/
 .nyc_output/
 coverage/
 doc/
-test/**/*.js
+test/unit/*.js
+test/integration/*.js
```

Comment:
Why have you done this? Before we ignored all javascript files in the test directory, now you want only to ignore js files if they are in either `/test/unit` or `test/integration`? I don't see any benefit to changing this

Reply:
It was when I created a .js file, I'll correct it now since I have the .ts script, not .js